Extension/src/main.ts

```diff
@@ -288,59 +288,36 @@ async function postInstall(info: PlatformInformation): Promise<void> {
 }
 
 async function finalizeExtensionActivation(): Promise<void> {
-    if (vscode.workspace.getConfiguration("C_Cpp", null).get<string>("intelliSenseEngine") === "Disabled") {
+    let settings: CppSettings = new CppSettings();
+    if (settings.intelliSenseEngine === "Disabled") {
         languageServiceDisabled = true;
         getTemporaryCommandRegistrarInstance().disableLanguageServer();
         disposables.push(vscode.workspace.onDidChangeConfiguration(() => {
-            if (!reloadMessageShown && vscode.workspace.getConfiguration("C_Cpp", null).get<string>("intelliSenseEngine") !== "Disabled") {
+            if (!reloadMessageShown && settings.intelliSenseEngine !== "Disabled") {
                 reloadMessageShown = true;
                 util.promptForReloadWindowDueToSettingsChange();
             }
         }));
         return;
     }
     disposables.push(vscode.workspace.onDidChangeConfiguration(() => {
-        if (!reloadMessageShown && vscode.workspace.getConfiguration("C_Cpp", null).get<string>("intelliSenseEngine") === "Disabled") {
+        if (!reloadMessageShown && settings.intelliSenseEngine === "Disabled") {
             reloadMessageShown = true;
             util.promptForReloadWindowDueToSettingsChange();
         }
     }));
     getTemporaryCommandRegistrarInstance().activateLanguageServer();
 
-    // Update default for C_Cpp.intelliSenseEngine based on A/B testing settings.
-    // (this may result in rewriting the package.json file)
-
-    let abTestSettings: cpptoolsJsonUtils.ABTestSettings = cpptoolsJsonUtils.getABTestSettings();
```

Comment:
Can you remove the corresponding deprecated properties from `./cpptools.json` as well?